pkg/runner/types.go

```diff
@@ -213,6 +213,11 @@ var routes = []routev1.Route{
 		ObjectMeta: metav1.ObjectMeta{
 			Namespace: benchmarkNs,
 			Name:      fmt.Sprintf("%s-passthrough", serverName),
+			Annotations: map[string]string{
+				// Passthrough terminations default balance strategy is source, this strategy is not suitable for
+				// performance testing when concurrency is higher than 1, as all the requests will use the same source IP
+				"haproxy.router.openshift.io/balance": "source",
```

Comment:
Can you update to `random` as it is the default strategy :)
```"haproxy.router.openshift.io/balance": "source",```

Reply:
Also, I believe, this needs a rebase :)

Reply:
fixed, I forgot the put it back when I was running some tests :)